package.json

```diff
@@ -59,6 +59,7 @@
     "gulp-template": "^4.0.0",
     "gulp-typescript": "^3.0.2",
     "gulp-uglify": "^1.5.3",
+    "gulp-tap": "^1.0.1",
```

Comment:
Use `npm i --save-dev some-package` to add a package. This line has an incorrect position and any `npm i` call will cause a sorting changes.